physionet-django/user/models.py

```diff
@@ -657,6 +658,7 @@ class Orcid(models.Model):
     token_type = models.CharField(max_length=50, default='', blank=True)
     token_scope = models.CharField(max_length=50, default='', blank=True)
     token_expiration = models.DecimalField(max_digits=50, decimal_places=40, default=0)
+    date_added = models.DateField(default=datetime(2200, 1, 1))
```

Comment:
In a previous version, I think you had something like `models.DateField(auto_now_add=True)`. Thought that seemed like a good idea. What was the reason to change?

Reply:
@bemoody , I think we are going to change this back to what I had previously: 
`date_added = models.DateField(auto_now_add=True)`
This means that new objects will be created based on the current timestamp whereas old objects will be derived from the token_expiration.